spec/open_weather/one_call/one_call_spec.rb

```diff
@@ -7,23 +7,40 @@
 
   it 'lat, lon', vcr: { cassette_name: 'one_call/lat_lon' } do
     data = client.one_call(lat: 33.441792, lon: -94.037689)
+
     expect(data).to be_a OpenWeather::Models::OneCall::Weather
     expect(data.lat).to eq 33.44
     expect(data.lon).to eq(-94.04)
     expect(data.timezone).to eq 'America/Chicago'
     expect(data.current).to be_a OpenWeather::Models::OneCall::CurrentWeather
+
+    # Minutely
     expect(data.minutely).to be_a Array
     expect(data.minutely.size).to eq 61
     expect(data.minutely.first).to be_a OpenWeather::Models::OneCall::MinutelyWeather
     expect(data.minutely.first.precipitation).to eq 0
+
+    # Hourly
     expect(data.hourly).to be_a Array
     expect(data.hourly.size).to eq 48
     expect(data.hourly.first).to be_a OpenWeather::Models::OneCall::HourlyWeather
     expect(data.hourly.first.temp).to eq 295.55
+
+    # Daily
     expect(data.daily).to be_a Array
     expect(data.daily.size).to eq 8
     expect(data.daily.first).to be_a OpenWeather::Models::OneCall::DailyWeather
     expect(data.daily.first.temp.night).to eq 293.67
+
+    # Alerts
```

Comment:
Record a new VCR and split this test, or re-record a new lat_lon.yml and update numbers in this one.

Reply:
Done - re-recorded in a location with current weather alerts.